[All Hosts] (devx) Split manifest conversion and Teams Toolkit articles (#5162)
OfficeDev/office-js-docs-pr: docs/develop/import-teams-toolkit.md

1. In the **Select import project manifest file** dropdown menu, browse to the add-in only manifest file, typically named **manifest.xml**.
1. In the **Workspace folder** dialog, select the folder where you want to put the converted project.
1. In the **Application name** dialog, give a name to the project (with no spaces). Teams Toolkit creates the project with your source files and scaffolding. It then opens the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.

### Verify that the add-in can be sideloaded

> [!NOTE]
> Add-ins that use the unified manifest can be sideloaded only on Office Version 2304 (Build 16320.20000) or later.

Sideload the add-in using the instructions at [Sideload with Teams toolkit](../testing/sideload-add-in-with-unified-manifest.md#sideload-with-the-teams-toolkit).

If you encounter problems, as a troubleshooting step, try [sideloading with a system prompt, bash shell, or terminal](../testing/sideload-add-in-with-unified-manifest.md#sideload-with-a-system-prompt-bash-shell-or-terminal). If you can, then the problem is isolated to the toolkit.

> It is possible that sideloading problems are the result of a file and folder structure, or configuration settings, that are different from what Teams Toolkit normally expects. See the section [Post importation: Adjust the project structure and settings as needed](#post-importation-adjust-the-project-structure-and-settings-as-needed).

Reviewer: ```suggestion
It's possible that sideloading problems are the result of a file and folder structure, or configuration settings, that are different from what Teams Toolkit normally expects. See the section [Post importation: Adjust the project structure and settings as needed](#post-importation-adjust-the-project-structure-and-settings-as-needed).
```

### Post importation: Adjust the project structure and settings as needed

The importation process creates some folders and files that Visual Studio Code or Teams Toolkit need, but it doesn't reorganize your source files; such as HTML, JavaScript, and CSS files. It also doesn't change the content of any files in the project, including tool configuration files. We recommend that you change your project to match the pattern of projects that are created in Teams Toolkit. As you work, keep the following points in mind.

- HTML files in the toolkit projects don't have inline `<script>` elements. They only use `<script>` elements with a `src` attribute that loads an external file.
- Source files in a new toolkit project are in a folder named **\src**. Within this folder, the source files are further divided into subfolders based on the runtimes in which they normally are run. The following is a typical structure.

>    ```command&nbsp;line

Reviewer: ```suggestion
   ```console
```
